### DISCUSSION TITLE
애니메이션 교체

### DISCUSSION THREAD

**Discussion by cinephile231** [naverz/zepeto-studio-kor#727] [5. Q&A]
Mixamo에서 총을 들고 있는 Idle, 총을 들고 Run
으로 기본 애니메이션을 교체하고 싶은데, 막막하네요.
일말의 단서라도 좀 알려주십시오.

**Comment by minjujuu** [comment]
https://github.com/naverz/zepeto-script-sample/issues/65
위 링크 참고하시면 기본 애니메이션 교체 하실 때 도움이 될 것 같습니다!

**Comment by smilejsu82** [comment]
> Mixamo에서 총을 들고 있는 Idle, 총을 들고 Run 으로 기본 애니메이션을 교체하고 싶은데, 막막하네요. 일말의 단서라도 좀 알려주십시오.

손에 총 붙이는것은 요고보시면 대구요 ~ 
https://github.com/naverz/zepeto-studio-kor/discussions/542

애니메이션 교체는 요고요 ~ 
https://github.com/naverz/zepeto-studio-kor/discussions/538

예시는 요고요 ~ 
https://github.com/naverz/zepeto-studio-kor/discussions/557

화이팅요 👍
